src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1686,6 +1686,16 @@ public void backticksQuotedIndexNameTest() throws Exception {
                 executeQuery("SELECT lastname FROM bank", "jdbc"),
                 executeQuery("SELECT `bank`.`lastname` FROM `bank`", "jdbc")
         );
+
+        assertEquals(
+                executeQuery(
+                        "SELECT `b`.`age` AS `hello world`, AVG(`b`.`balance`) FROM `bank` AS `b` " +
+                                "WHERE ABS(`b`.`age`) > 20 GROUP BY `b`.`gender` ORDER BY `b`.`age`",
+                "jdbc"),
+                executeQuery("SELECT age , AVG(balance) FROM bank " +
```

Comment:
Because field `age` doesn't have the alias (hello world), could you confirm if the assertion works as expected?